### PR TITLE
Update changelog with announcement on OSM 1.10 features

### PR DESCRIPTION
**Pull request by iroussos**
# Description

Update changelog with announcement on OSM 1.10 features

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

**Comment by iroussos** [comment]
@billy-the-fish added a section on the changelog with the announcement about the features introduced with OSM 1.10. That one got lost and was never added 2 weeks ago, so I am retroactively adding it back to the day it should have been added. If you don't see any big issues, feel free to make any changes you want and merge

File: about/changelog.md [hunk 1]
```diff
@@ -110,6 +110,17 @@ Navigate to the AI tab in your service overview and follow the instructions to a
 ### PostgreSQL-to-PostgreSQL foreign data wrappers: 
 Fetch and query data from multiple PostgreSQL databases, including time-series data in hypertables, directly within Timescale Cloud using [foreign data wrappers (FDW)](https://docs.timescale.com/use-timescale/latest/schema-management/foreign-data-wrappers/). No more complicated ETL processes or external tools—just seamless integration right within your SQL editor. This feature is ideal for developers who manage multiple PostgreSQL and time-series instances and need quick, easy access to data across databases.
 
+### Faster queries over Tiered Data 
+This release adds support for runtime chunk exclusion for queries that need to access tiered storage. Chunk exclusion now works with queries that use stable expressions in the WHERE clause. Most common form of this type of query is:
```
Comment (billy-the-fish) [inline about/changelog.md]:
```suggestion
This release adds support for runtime chunk exclusion for queries that need to access [tiered storage](https://docs.timescale.com/use-timescale/latest/data-tiering/). Chunk exclusion now works with queries that use stable expressions in the `WHERE` clause. The most common form of this type of query is:
```

File: about/changelog.md [hunk 2]
```diff
@@ -110,6 +110,17 @@ Navigate to the AI tab in your service overview and follow the instructions to a
 ### PostgreSQL-to-PostgreSQL foreign data wrappers: 
 Fetch and query data from multiple PostgreSQL databases, including time-series data in hypertables, directly within Timescale Cloud using [foreign data wrappers (FDW)](https://docs.timescale.com/use-timescale/latest/schema-management/foreign-data-wrappers/). No more complicated ETL processes or external tools—just seamless integration right within your SQL editor. This feature is ideal for developers who manage multiple PostgreSQL and time-series instances and need quick, easy access to data across databases.
 
+### Faster queries over Tiered Data
```
Comment (billy-the-fish) [inline about/changelog.md]:
```suggestion
### Faster queries over tiered data 
```

File: about/changelog.md [hunk 3]
```diff
@@ -110,6 +110,17 @@ Navigate to the AI tab in your service overview and follow the instructions to a
...
+### Faster queries over Tiered Data 
+This release adds support for runtime chunk exclusion for queries that need to access tiered storage. Chunk exclusion now works with queries that use stable expressions in the WHERE clause. Most common form of this type of query is:
+
+```
+SELECT * FROM  hypertable WHERE timestamp_col > now() - '100 days'::interval
+```
+
+For more info on queries with immutable/stable/volatile filters, check our blog post on [Implementing constraint exclusion for faster query performance](https://www.timescale.com/blog/implementing-constraint-exclusion-for-faster-query-performance/).
+
+Additionally, there is now an option to disable tiering on a hypertable. If you no longer want to use tiered storage for a particular hypertable, you can drop the associated metadata by calling the [disable_tiering function](https://docs.timescale.com/use-timescale/latest/data-tiering/disabling-data-tiering/).
```
Comment (billy-the-fish) [inline about/changelog.md]:
```suggestion
If you no longer want to use tiered storage for a particular hypertable, you can now disable tiering and drop the associated tiering metadata on the hypertable with a call to [disable_tiering function](https://docs.timescale.com/use-timescale/latest/data-tiering/disabling-data-tiering/). 
```

**Review by billy-the-fish — APPROVED** [review]
Couple of language suggestions.

**Pull request merged**